tests/docs/writing-integration-test.md

```diff
@@ -5,14 +5,21 @@
 Integration tests are run against a live running daprd binary locally. Each test
 scenario is run against a new instance of daprd, where each scenario modifies
 the daprd configuration to suit the test. Tests are expected to complete within
-seconds, ideally less than 5, and should not take longer than 30. Daprd is
+seconds, ideally less than 5, and should not take longer than 30. Binaries
 always built from source within the test.
 
 
 ## Invoking the test
 
 ```bash
-go test -v -race -count -tags="integration" ./tests/integration` -run="Test_Integration/daprd/pubsub/http/fuzzpubsubNoRaw"
+go test -v -race -tags integration ./tests/integration
+```
+
+You can also run a subset of tests by specifying the `-focus` flag, which takes a [Go regular expression](https://github.com/google/re2/wiki/Syntax).
+
+```bash
+go test -v -race -tags integration ./tests/integration -focus sentry
+go test -v -race -tags integration ./tests/integration -test.skip Test_Integration/sentry/validator/jwks -focus sentry
```

Comment:
I am confused by this example, as there's a `-test.skip` and `-focus`

How should focus be used?

Reply:
I have updated this page with comments. The skip example here means to run all sentry related tests, but skip this specific jwks validator tes .